src/software/thunderscope/binary_context_managers/binary_context_managers.py

```diff
@@ -0,0 +1,24 @@
+import psutil as util
```

Comment:
bump: you should rename this file to utils.py

Reply:
I think that is a external dependency: 

see 

https://github.com/Mr-Anyone/Thunderbot_Software/blob/binary_context_managers_refactor/environment_setup/ubuntu22_requirements.txt#L13

Reply:
Arun is referring to the file being renamed, not the import. Since the file has nothing to do with "binary context management" anymore.

Reply:
I see, my bad.

Reply:
done